### PR TITLE
DL-2663 - Updated Dependencies

### PR DESCRIPTION
**Pull request by alijy**
# DL-2663 - Updated dependencies

**New feature**

Updated dependencies

## Checklist

 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [x]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [x]  I've run a dependency check to ensure all dependencies are up to date

## Reviewer (add name here)

 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [ ]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

**Comment by alijy** [comment]
> org.scoverage:sbt-scoverage can be upgraded further

Done

**Comment by stephen-oppong-beduh** [comment]
The build.sbt file is name ```buid.sbt```

File: buid.sbt [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+lazy val microservice = Project(appName, file("."))
+  .enablePlugins(Seq(play.sbt.PlayScala) ++ plugins : _*)
+  .settings(playSettings ++ scoverageSettings : _*)
+  .settings(PlayKeys.playDefaultPort := 9372)
+  .settings(majorVersion := 1)
+  .settings(publishingSettings : _*)
+  .settings(
+    targetJvm := "jvm-1.8",
+    scalaVersion := "2.11.12",
```
Comment (stephen-oppong-beduh) [inline buid.sbt]: The scala version is set as 2.11.12, but the build-job has it as 2.12.x. It's also set in multiple places

Comment (alijy) [reply buid.sbt]: There were 2 copies of scalaVersion both set to 2.11.12. Removed the other and upgraded to 2.12.10

**Pull request merged**